services/packagist/packagist-stars.tester.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const { isMetric } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Stars (valid package)')
+  .get('/guzzlehttp/guzzle.json')
+  .expectBadge({
+    label: 'stars',
+    message: isMetric,
+  })
+
+t.create('Stars (invalid package)')
+  .get('/frodo/is-not-a-package.json')
+  .expectBadge({
+    label: 'stars',
+    message: 'not found',
+  })
+
+t.create('Stars (valid package, valid custom server)')
+  .get('/guzzlehttp/guzzle.json?server=https%3A%2F%2Fpackagist.org')
+  .expectBadge({
+    label: 'stars',
+    message: isMetric,
+  })
+
+t.create('Stars (invalid package, valid custom server)')
```

Comment:
This test seems to be the same as above, the package doesn't seem invalid.

Reply:
Good catch. Will fix it.

Reply:
Same in line 42.